Resources/Prototypes/Entities/Clothing/Head/soft.yml

```diff
@@ -239,3 +239,25 @@
     sprite: Clothing/Head/Soft/yellowsoft_flipped.rsi
   - type: Clothing
     sprite: Clothing/Head/Soft/yellowsoft_flipped.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatBizarreSoft
+  name: troublemaker's soft
```

Comment:
soft what

Reply:
The cap, that's how all caps are named and defined in yaml